src/styles/blocks/button.scss

```diff
@@ -0,0 +1,31 @@
+.button {
+  height: 56px;
+  width: 100%;
+  padding-inline: 48px;
+  background-color: $main-text-color;
+  color: #fff;
```

Comment:
for large projects, it is better to put all colors in variables, even if they are used 1 time